### DISCUSSION TITLE
How to launch the GUi?

### DISCUSSION THREAD

**Discussion by gendalv** [lshqqytiger/stable-diffusion-webui-directml#158] [Q&A]
webui-user.bat has downloaded and installed stuff.
but, no actual GUI is launching. So now what?
Windows 11, 6600xt

ok figured out that I have to actually open the link in browser from this https://www.youtube.com/watch?v=E3aUPAXrV_8
thought it would launch a UI itself.

**Comment by Alexmofu** [comment]
Take a look on what you've copied here:

Running on local URL:  [http://127.0.0.1:7860](http://127.0.0.1:7860)

You can also add `--autolaunch` into [COMMANDLINE_ARGS](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings) so it opens automatically

**Comment by gendalv** [comment]
my COMMANDLINE_ARGS now are: --autolaunch --medvram --precision full --no-half --opt-sub-quad-attention --opt-split-attention-v1 
but I'm still getting vram error sometimes.

Which samplers are the best?
